### PR TITLE
Fix docs json files

### PR DESCRIPTION
**Pull request by maddieclayton**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
netsign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/344/

File: src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1 [hunk 1]
```diff
@@ -147,8 +147,7 @@ CmdletsToExport = 'Remove-AzAvailabilitySet', 'Get-AzAvailabilitySet',
     'New-AzDiskConfig', 'Set-AzDiskDiskEncryptionKey', 
     'Set-AzDiskImageReference', 'Set-AzDiskKeyEncryptionKey', 
     'New-AzDiskUpdateConfig', 
-    'Set-AzDiskUpdateDiskEncryptionKey', 
-    'Set-AzDiskUpdateImageReference',
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1]: what happened to these imagereference cmdlets?

**Comment by maddieclayton** [comment]
@azuresdkci Test this please

File: src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1 (hunk 1 above)
Comment (maddieclayton) [reply src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1]: They appear to have been taken out a while ago in AzureRM but were never removed in Az.  I caught this when trying to generate the index.json file, as the help files/cmdlet code was removed when the cmdlets were taken out of AzureRM.

**Pull request merged**